CHANGELOG.md

```diff
@@ -543,14 +552,18 @@ Additions to existing modules
   nonZeroDivisor : DivMod dividend divisor → NonZero divisor
   ```
 
-* Added new proofs in `Data.Nat.Properties`:
+* Added new proofs and 'guarded' version of `_∸_` in `Data.Nat.Properties`:
```

Comment:
I think this comment both here and in the main file can be removed?

Reply:
Or even (Fairbairn again?) remove altogether?
If we don't have it as a definition, and the proof of this property is definitionally `refl`, maybe it's actually redundant?